### DISCUSSION TITLE
Summary of changes

### DISCUSSION THREAD

**Discussion by thevops** [helmfile/helmfile#965] [Ideas]
Hello,

Helmfile is a great tool. I love it! Thanks for your work on this. 
It's pretty similar to Terraform which I love too. I want to propose a feature.

It'd be great to see the summary of changes at the end of the diff. Something like the Terraform has.
It is for Terraform:
```
Plan: 3 to add, 2 to change, 1 to destroy.
```

It'd be even better to display all files one by one with the status. Something like:
```
my-chart/template/configmap.yaml REMOVED
my-chart/template/pod.yaml CREATED
my-chart/template/secret.yaml MODIFIED
```

It is something like the current way for releases:
```
Affected releases are:
  my-chart (./charts/my-chart) UPDATED

Do you really want to apply?
  Helmfile will apply all your changes, as shown above.
```

But my suggestion is for files inside each chart (release).

What do you think? Maybe there is such a solution but I don't know about it 😅

**Comment by yxxhero** [comment]
@thevops good idea. PR is welcome.

**Reply by thevops** [reply]
Hey, I've found the flag `--args '--output simple'`. It shows the summary like Terraform.
Moreover, there is also the flag `--context X` - it shows changes with X lines of context instead of a full manifest.

So there is no need for the PR described above :) I'm closing this discussion
